src/plugins/CMakeLists.txt

```diff
@@ -34,6 +34,7 @@ function(add_plugin)
 endfunction(add_plugin)
 
 add_plugin(NAME hyriseMvccDeletePlugin SRCS mvcc_delete_plugin.cpp mvcc_delete_plugin.hpp DEPS sqlparser magic_enum gtest)
+add_plugin(NAME hyriseSecondTestPlugin SRCS second_test_plugin.cpp second_test_plugin.hpp DEPS sqlparser)
```

Comment:
Second one so that we can have better tests: testing name clashes of user callable functions originating from different plugins. 

Second before first because it is ordered alphabetically.